### DISCUSSION TITLE
Open stream from twitch.tv on Streamlink Twitch GUI

### DISCUSSION THREAD

**Discussion by jpdsc** [streamlink/streamlink-twitch-gui#950] [General]
Is it possible to click on a stream on twitch.tv but it will instead open on Streamlink Twitch GUI?
Browsing on twitch.tv still works much better then on the Streamlink Twitch GUI

**Comment by bastimeyer** [comment]
I suggest you read the readme, so you understand the purpose of Streamlink Twitch GUI, and most importantly the difference between Streamlink and Streamlink Twitch GUI.

**Reply by jpdsc** [reply]
I understand the purpose and I have everything set-up :-)
I was just wondering if it would be possible when browsing twitch.tv and clicking on a stream it would instead open it in Streamlink Twitch GUI.

Anyway, I don't think it's possible but thanks anyway for replying.
